dkim/verify.go

```diff
@@ -86,11 +86,19 @@ type signature struct {
 	v string
 }
 
+type VerifierOptions struct {
+	LookupTXT txtLookupFunc
```

Comment:
We can't use unexported types (`txtLookupFunc`) in exported structs (`VerifierOptions`) like this. What about spelling out the function signature here to make it easy to read in GoDoc?

Would be nice to add a little documentation comment too.

Reply:
fixed